app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartController.scala

```diff
@@ -69,6 +69,9 @@ class StartController @Inject()(
       case (_, Some(_: Subscribed)) =>
         Redirect(routes.HomeController.homepage())
 
+      case (_, Some(AlreadySubsribedWithDifferentGGAccount)) =>
```

Comment:
typo in  `AlreadySubsribedWithDifferentGGAccount`

Reply:
fixed